indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
Fine by me then. Even though toolbars are not optimized for being wrapped, truth is they do get wrapped on smaller screens or no fullscreen windows. This way at least doesn't look broken.